packages/common/src/ceramic-api.ts

```diff
@@ -76,7 +77,7 @@ export interface CeramicApi {
      * @param docId - Document ID
      * @param opts - Initialization options
      */
-    loadDocument<T extends Doctype>(docId: DocID | string, opts?: DocOpts): Promise<T>;
+    loadDocument<T extends Doctype>(docId: DocID | string, opts?: DocOpts, tip?: CID): Promise<T>;
```

Comment:
Adding a tip arg when there's already an opts arg feels weird to me. It makes me feel like we should probably split `DocOpts` into `DocWriteOpts` and `DocLoadOpts`. It's already the case that the `anchor` and `publish` flags are meaningless to read operations, and the `sync` flag is mostly meaningless for write operations. @oed @PaulLeCam

Reply:
Yes I totally agree, let's use different options types for read and write, and include any needed flag in these options 👍

Reply:
Yeah, makes sense to just put the flag into the `opts`.